api/src/main/kotlin/eu/codeloop/thehub/jobs/sync/JobFetcher.kt

```diff
@@ -14,10 +16,14 @@ class JobFetcher(private val jobFetcher: JobApiFetcher, private val jobHtmlFetch
         return jobsFromApi.mapNotNull { api -> fetch(host, api) }
     }
 
+    @Value("\${thehub.models.JobEntity.approved_at}")
+    private lateinit var approvedAtDefault: String
```

Comment:
Note: if you get properties from `application.yml` it is good idea to introduce property class

```
@ConfigurationProperties(prefix = "app.foo")
data class FooProperties(
    var bar: String
)
```

Please remember that you need to use `@EnableConfigurationProperties(FooProperties::class)`

Reply:
I have problem with injecting property class. Could look at 8c61de1f65ac0847f6ec7be9b62cdf207796f913?